nwbwidgets/ophys/ophys_controllers.py

```diff
@@ -0,0 +1,97 @@
+from typing import Optional
+
+import ipywidgets as widgets
+
+from ..controllers import RotationController, ImShowController, ViewTypeController, MultiController
+
+
+class FrameController(widgets.VBox):
+    controller_fields = ("frame_slider",)
```

Comment:
maybe this should be a dictionary, which maps a class attribute to an internal widget or controller value. e.g.

```python
controller_fields = dict(frame: "frame_slider")
```

would imply that frame_controller.frame gives you the `value` of `frame_controller.frame_slider`.